account_invoice_purchase_copy_to_comment_field/models/account_invoice.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    origin = fields.Char(
+        string='Source Document',
```

Comment:
Remove string and help when they are not changed

Reply:
They have been removed now